AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

```diff
@@ -37,6 +37,8 @@ craftscouncil.org.uk#@#.cookie-banner:not(body):not(html)
 @@||api.usercentrics.eu/translations/
 ! https://github.com/AdguardTeam/AdguardFilters/issues/183565
 @@||panini.com.br/*/Amasty_GdprFrontendUi/
+! https://github.com/AdguardTeam/AdguardFilters/issues/184460
+heelys.eu.com##.pop-cookie-overlay
```

Comment:
Please move this rule to `cookies_specific.txt` - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_specific.txt
It should added in `Regular rules` section, somewhere here: https://github.com/AdguardTeam/AdguardFilters/blob/91f4cd784c72835252d90844ec06f3970e4e6920/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L278-L280